File: Brute Force/selectionsort.py
# Selection Sort
# code by Akshat Aryan
import numpy as np

def selectionsort(arr):
    for i in range(0, len(arr) - 1):
        min = i
        for j in range(i + 1, len(arr)):
            if(arr[j] < arr[min]):
                min = j
        (arr[i], arr[min]) = (arr[min], arr[i])

if(__name__ == "__main__"):
    arr = np.array(list(map(int, input("Enter the array:").split())))
    selectionsort(arr)
    print("Sorted array:\n", arr)

File: Brute Force/selectionsort.c
// Selection Sort
// code by Akshat Aryan
#include <stdio.h>

void swap(int *a, int *b){
    int temp = *a;
    *a = *b;
    *b = temp;
}

void selectionsort(int arr[], int n){
    int min;
    for(int i = 0; i < n - 1; i++){
        min = i;
        for(int j = i + 1; j < n; j++){
            if(arr[j] < arr[min]){
                min = j;
            }
        }
        swap(&arr[i], &arr[min]);
    }
}

int main(){
    int n;
    printf("Enter the number of elements in the array:");
    scanf("%d", &n);
    int arr[n];
    printf("Enter the array:");
    for(int i = 0; i < n; i++){
        scanf("%d", &arr[i]);
    }
    selectionsort(arr, n);
    printf("Sorted array is:\n");
    for(int i = 0; i < n; i++){
        printf("%d ", arr[i]);
    }
    return 0;
}
